spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
+conservation;protected_area;nature_reserve;urban;rural;residential;forest;wood;nature_reservefarm;farmland,,>=13,>=100000,2,13
+conservation;protected_area;nature_reserve;urban;rural;residential;forest;wood;nature_reservefarm;farmland,,>=14,>=50000,2,14
+conservation;protected_area;nature_reserve;urban;rural;residential;forest;wood;nature_reservefarm;farmland,,>=15,>=2000,2,15
+conservation;protected_area;nature_reserve;urban;rural;residential;forest;wood;nature_reservefarm;farmland,,>=16,+,2,16
+aerodrome;military;university;college,,>=8,>=10000000,3,8
+aerodrome;military;university;college,,>=9,>=5000000,3,9
+aerodrome;military;university;college,,>=10,>=1000000,3,10
+aerodrome;military;university;college,,>=11,>=500000,3,11
+aerodrome;military;university;college,,>=12,>=500000,3,12
+aerodrome;military;university;college,,>=13,>=100000,3,13
+aerodrome;military;university;college,,>=14,>=50000,3,14
+aerodrome;military;university;college,,>=15,>=2000,3,15
+aerodrome;military;university;college,,>=16,+,3,16
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=10,>=1000000,4,10
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=11,>=500000,4,11
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=12,>=500000,4,12
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=13,>=100000,4,13
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=14,>=50000,4,14
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=15,>=2000,4,15
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=16,+,4,16
+theme_park;resort;aquarium;winery;maze;beach,,>=10,>=1000000,5,10
```

Comment:
Bubble Wrap is confused about `glacier`. It's commented out in the final Tier 6 section https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6534-L6541 that includes `garden`. The labels section for `glacier` starts at zoom 10 on  https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L5141. Let's include it with Tier 5 here and we'll update the scene files.